CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.11.0] - 2022-04-21
+
+v0.11.0 is a release that reworks how subscriptions are handled by the servers where the users have to explicitly reject or accept each subscription.
+The reason for this is that the actual params in subscription is passed to the callback and it might fail which the server can't know when looking at the call.
```

Comment:
```suggestion
The reason for this is that the actual params in the subscription is passed to the callback and it might fail dependent on the application which the server can't know when looking at the call.
```